languages/csharp/exercises/concept/constructors/Constructors.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+
+class RemoteControlCar
+{
+    // TODO: define the 'RemoteControlCar' classes' constructor
+
+    public bool BatteryDrained()
+    {
+        throw new NotImplementedException("Please implement the RemoteControlCar.BatteryDrained() method");
+    }
+
+    public int DistanceDriven()
+    {
+        throw new NotImplementedException("Please implement the RemoteControlCar.DistanceDriven() method");
+    }
+
+    public void Drive()
+    {
+        throw new NotImplementedException("Please implement the RemoteControlCar.Drive() method");
+    }
+
+    public static RemoteControlCar TopModel()
+    {
+        throw new NotImplementedException("Please implement the (static) RemoteControlCar.TopModel() method");
+    }
+}
+
+class RaceTrack
+{
+    // TODO: define the 'RaceTrack' classes' constructor
```

Comment:
```suggestion
    // TODO: define the 'RaceTrack' class's constructor
```

Alternatively, consider switching to:
```suggestion
    // TODO: define the constructor for this class
```

Reply:
I want to explicitly mention the class name to me it as unambiguous as possible.